### PR TITLE
Tidy: Fix ordering use statements with braces

### LINKED ISSUE #7412: Import ordering tidy check confused by multiple use statements from same module
```rust
use url::{Url, UrlParser, SchemeData};
use url::urlutils::{UrlUtils, UrlUtilsWrapper};
```
```
./components/script/dom/urlhelper.rs:7: use statement is not in alphabetical order
./components/script/dom/urlhelper.rs:8: use statement is not in alphabetical order
```
<hr>
```rust
use url::{Url, UrlParser};
use url::SchemeData;
```
```
./components/script/dom/urlhelper.rs:9: use statement is not in alphabetical order
./components/script/dom/urlhelper.rs:10: use statement is not in alphabetical order
```

### PR DESCRIPTION
**Pull request by UK992**
This hack fixes https://github.com/servo/servo/issues/7412 and matches behavior with rustfmt.

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/13205)

### CODE REVIEW

**Comment by Wafflespeanut** [comment]
It's actually a nice hack, so to speak. Will look at the entire PR later today! :)

File: python/tidy/servo_tidy/tidy.py [hunk 1]
```diff
@@ -516,7 +516,10 @@ def check_rust(file_name, lines):
                 yield (idx + 1, "use statement spans multiple lines")
             # strip "use" from the begin and ";" from the end
             current_use = line[4:-1]
-            if indent == current_indent and prev_use and current_use < prev_use:
+            if prev_use:
+                current_use_cut = current_use.replace("{self,", ".").replace("{", ".")
+                prev_use_cut = prev_use.replace("{self,", ".").replace("{", ".")
+            if indent == current_indent and prev_use and current_use_cut < prev_use_cut:
```
Comment (Wafflespeanut) [inline python/tidy/servo_tidy/tidy.py]: Maybe just merge this into the previous block as

``` python
if indent == current_indent and current_use_cut < prev_use_cut:
```

**Comment by Wafflespeanut** [comment]
The changes look good! r+ after addressing my comment. And, while you were at it, can you please change your commit message? "Fix tidy" looks rather confusing, since it doesn't actually fix tidy. It fixes the sorting of `use` statements reported by tidy :smile:

File: python/tidy/servo_tidy/tidy.py (hunk 1 above)
Comment (UK992) [inline python/tidy/servo_tidy/tidy.py]: done

**Comment by UK992** [comment]
done.

**Comment by Wafflespeanut** [comment]
Thanks! :)

@bors-servo r+

**Comment by Wafflespeanut** [comment]
Sorry I didn't prioritize this first. Please rebase it this once, and I'll make sure it lands first! :)

**Comment by UK992** [comment]
rebased.

**Comment by Wafflespeanut** [comment]
[Looks like this is next](http://build.servo.org/homu/queue/servo) to #13193. We can only hope that the other PR doesn't break yours.

**Comment by Wafflespeanut** [comment]
Yay! There we go! :smile:

**Comment by Wafflespeanut** [comment]
Now, we've broken everything :neutral_face:

**Pull request merged**